news/update_dockerfile.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:**
+
+* Change the repo from cnerg/pyne to pyne/pyne
```

Comment:
I think you are missing the openMC integration into the dockerfile

Reply:
Add OpenMC python API information in 7efd6bc